src/Utilities/FractionUtilities.hpp

```diff
@@ -69,25 +67,26 @@ class ContinuedFraction {
 
   /// Advance to the next element in the expansion
   ContinuedFraction& operator++() noexcept {
-    if (remainder_ == 0 or (error_ /= square(remainder_)) > 1) {
+    // Terminate when remainder_ is consistent with zero.
+    if (remainder_ == 0 or error_ > remainder_) {
```

Comment:
Note to self: Check whether `error_` should be initialized relative to the input magnitude, and see if that matters here if so.

Reply:
It should have been.  I have added a fixup changing it.